app/assets/javascripts/common/reports.js

```diff
@@ -1106,16 +1103,15 @@ function baseLineGraphConfig() {
               family: "Roboto",
             },
             padding: 6,
+            showLabelBackdrop: true,
```

Comment:
upgrading to v4.2.1 moves the dates on the x axis to the left slightly.
Seems like they line up better when dashes are shown on the x-axis.
Adding this re-aligns the labels back to where they were positioned in v3.9.1
![Screenshot 2023-03-21 at 11 20 55](https://user-images.githubusercontent.com/9541902/226643419-c0d4c546-2e23-4f57-bd80-73dd06901f48.png)
![Screenshot 2023-03-21 at 11 20 13](https://user-images.githubusercontent.com/9541902/226643431-be2851ea-9a08-4704-bb9c-0b319bdf1ecd.png)

I've submitted an error report about this - will keep an eye on this.
https://github.com/chartjs/Chart.js/issues/11197